server/graphql/v2/mutation/TierMutations.ts

```diff
@@ -123,7 +123,11 @@ const tierMutations = {
       await twoFactorAuthLib.enforceForAccountAdmins(req, account, { onlyAskOnLogin: true });
 
       // Create tier
-      const tier = await TierModel.create({ ...transformTierInputToAttributes(args.tier), CollectiveId: account.id });
+      const tier = await TierModel.create({
+        ...transformTierInputToAttributes(args.tier),
+        CollectiveId: account.id,
+        currency: account.currency,
```

Comment:
At some point in the future, it's possible we'll let Tier be defined in a different currency.

It's already supported by the API.

https://github.com/opencollective/opencollective/issues/3891

Reply:
Just a FYI, we don't have to make it work here now.